### ISSUE TITLE
[Git `HEAD`] Broken `common.mak`?

### ISSUE THREAD

**Issue by hartwork** [linux-test-project/lcov#368]
Hi!

This output in the [GCC 9 test log](https://github.com/hartwork/lcov-fork/actions/runs/12444996626/artifacts/2351716347) (of run https://github.com/hartwork/lcov-fork/actions/runs/12444996626) caught my attention:

```
[..]
======================================================================
DATE .......: 2024-12-21 12:07:21 +0000
TESTNAME ...: lcov/misc/help.sh
COMMAND ....: "/home/runner/work/lcov-fork/lcov-fork/tests/lcov/misc/help.sh  *"
OUTPUT .....: 
  ../../common.mak: line 3: MAKEFILE_LIST: command not found
  ../../common.mak: line 3: lastword: command not found
  realpath: missing operand
  Try 'realpath --help' for more information.
  ../../common.mak: line 3: TOPDIR: command not found
  ../../common.mak: line 5: MAKEFILE_LIST: command not found
  ../../common.mak: line 5: firstword: command not found
  realpath: missing operand
  Try 'realpath --help' for more information.
  ../../common.mak: line 5: TESTDIR: command not found
  ../../common.mak: line 7: syntax error near unexpected token `$(LCOV_HOME),'
  ../../common.mak: line 7: `ifeq ($(LCOV_HOME),)'
[..]
```

The two questions to me are "why is this happening" and "how is 'make check' not failing as a result"?

**Comment by henry2cox** [comment]
> The two questions to me are "why is this happening" and "how is 'make check' not failing as a result"?

Typo (including wrong file in shellscript), and ran to completion successfully because the shellscript didn't check that the file was sourced correctly - and didn't actually depend on anything in the file.

Fixed in 3b6caeb.

**Comment by hartwork** [comment]
@henry2cox thanks for the fix! :pray:
